docs/best_practices.rst

```diff
@@ -0,0 +1,140 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+==============
+Best Practices
+==============
+
+Websites you design and build for Mozilla as a consultant should adhere 
+to the same set of best practices used by Mozilla for its own websites. 
+Here are key practices you should follow throughout a project:
+
+
+Progressive Enhancement
+-----------------------
+
+With few exceptions, Mozilla websites should practice `progressive enhancement`_.
+Begin with simple, meaningful HTML. Add CSS and JavaScript as progressive
+layers in a way that still falls back to usable, accessible content in older
+browsers that don't support the latest features or in the event of JavaScript
+failing.
+
+.. _`progressive enhancement`: http://alistapart.com/article/understandingprogressiveenhancement
+
+Don't rely exclusively on JavaScript to serve static content. This is less 
+about catering to the few people who purposely disable JavaScript and more 
+about ensuring some fault tolerance as a best practice. While single-page 
+websites that render the entire screen using client-side JavaScript may have 
+a place, the reality is that 99% of the time they don't need to be built in 
+such a way. If a JavaScript error occurs or someone is using an older web 
+browser, their experience should degrade gracefully. No one should be greeted 
+by an empty, white screen.
+
+
+Feature detection
+-----------------
+
+Going hand-in-hand with progressive enhancement, Mozilla also strongly advocates 
```

Comment:
Suggest: Make this stronger we want our sites build using progressive enhancement. "Mozilla websites should be built using feature detection."